config-windows.md

```diff
@@ -112,11 +112,30 @@ For more information about tooling to generate a gMSA, see [Deployment Overview]
 When a container terminates, the Host Compute Service indicates if a Windows update servicing operation is pending.
 You can indicate that a container should be started in a mode to apply pending servicing operations via the OPTIONAL `servicing` field of the Windows configuration.
 
-
 ### Example
 
 ```json
     "windows": {
         "servicing": true
     }
+```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container MUST be run with Hyper-V isolation. If omitted, the container MUST be run as a Windows Server container.
+
+The following parameters can be specified:
+
+* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM. If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
```

Comment:
@jhowardmsft Thanks for the background info.  Your call on this.  If I was personally making the decision, I'd put in the sample that you had in your comment, it at least gets the idea into the users mind.  I would ask though that once a real world sample can be given here that you loop back and either edit or add it.  Cheers!